views/js/pciCreator/dev/audioRecordingInteraction/runtime/js/providers/webAudio.js

```diff
@@ -83,12 +83,13 @@ define([
          * The provider
          */
         webAudioProvider = {
+
             /**
              * Expose the Audio context so consumers can use it instead of creating a new one
              * @returns {AudioContext}
              */
             getAudioContext: function getAudioContext() {
-                return audioContext;
+                return audioContext || window.audioContext;
```

Comment:
The `audioContext` variable can be removed as you already removed its initialization.